tools/Generator.CLI/SourceGenerator/TaskTupleAwaiterGenerator.cs

```diff
@@ -0,0 +1,196 @@
+using System.Text;
+using static Generator.CLI.SourceGenerator.CommonTokens;
+
+namespace Generator.CLI.SourceGenerator
+{
+	public static class TaskTupleAwaiterGenerator
+	{
+		public static string GenerateContent(int arity)
+		{
+			/**
+			 * based on https://github.com/buvinghausen/TaskTupleAwaiter/blob/master/src/TaskTupleAwaiter/TaskTupleExtensions.cs
+			 * https://github.com/buvinghausen/TaskTupleAwaiter/commit/ead4851edaf7553fc938a22377e48ff9f5c70cdc
```

Comment:
Single permalink? https://github.com/buvinghausen/TaskTupleAwaiter/blob/ead4851edaf7553fc938a22377e48ff9f5c70cdc/src/TaskTupleAwaiter/TaskTupleExtensions.cs

![image](https://user-images.githubusercontent.com/8040367/51803489-bc4eee80-2223-11e9-8a59-7b139e610904.png)

Reply:
What is it you want me to do here? Add a permalink to the readme with a permalink to the old version?

Reply:
No, I don't see where the readme is involved. 😕 Your first link will break if we move files around, so it should be a permalink like the one I mentioned. I didn't understand the significance of the second link. Did you mean to link to a diff?

Reply:
Ahhh ok. Now i get what you mean. The intend was to have a pointer to the original file. What do you prefer? Permalink, or no link at all?

Reply:
Permalink sounds good. Will the one I initially suggested work fine?